src/component/manage_my_tariffs.js

```diff
@@ -18,8 +18,17 @@ export default class ManageMyTariffs extends ViewBase{
     this.filterText = "";
   }
 
+	noTariffsBannerTemplate() {
+		return html`<div class="w3-panel w3-round w3-light-grey">
+			<p>${this.t("manageMyTariffsBanner")}</p>
```

Comment:
This should be `manageMyTariffsBannerMessage`

Reply:
Fixed.